content/en/tracing/guide/metrics_namespace.md

```diff
@@ -103,7 +103,7 @@ With the following definitions:
 
 `trace.<SPAN_NAME>.duration`
 : **Prerequisite:** This metric exists for any APM service.<br>
-**Description:** Measure the total time for a collection of spans. Specifically, it is the total time spent by all spans over an interval - including time spent waiting on child processes.<br>
+**Description:** Measure the total time for a collection of spans. Specifically, it is the total time spent by all spans over an interval - including time spent waiting on child processes. When divided by `trace.<SPAN_NAME>.hits`, the result provides the average. To get percentiles, use [DDSketch-based metrics][11]. <br>
```

Comment:
@wantsui while factually correct, we should guide users to use the sketch metrics always, not just for percentiles. A sketch metric can be aggregated with avg:trace.<op_name> as like it can be aggregated with p99:trace.<op_name>.


Maybe something like: When divided by `trace.<SPAN_NAME>.hits` can yield an average latency but we recommend using [DDSketch-based metrics][11] for all latency calculations.

Reply:
Done on the latest commit!

Should a similar message be applied to all the duration metrics written under **Duration by** as well @omris1 ?

Reply:
I'm marking this resolved since we discussed the wording in a separate thread. The wording now marks this metric as legacy and recommends using DDSketch.